raiden/storage/sqlite.py

```diff
@@ -76,6 +89,23 @@ def _filter_from_dict(current: Dict[str, Any]) -> Dict[str, Any]:
     return filter_
 
 
+def _query_to_string(query: Query):
```

Comment:
Can you add a docstring to what this function does, as it seems to be the only function that deals with the `Query` NamedTuple?